numa_streams/numa_streams_extra/run_numa_stream

```diff
@@ -196,15 +208,18 @@ install_numa_streams()
 {
 	dnf install wget gcc gcc-c++ autoconf hwloc hwloc-gui libomp  -y
 	git clone https://github.com/jemalloc/jemalloc.git
-	pushd jemalloc
+	pushd jemalloc > /dev/null
 	./autogen.sh
 	make
 	make install
-	popd
+	popd > /dev/null
 
 	mkdir /opt/AMD
 	touch /opt/AMD/setenv_AOCC.sh
 
+	#
+	# We are getting the test from the phoronix git.
```

Comment:
Are we really getting it from a git repository?  This just seems like a tarball phoronix hosts